src/components/paginator-render/paginator-render.tsx

```diff
@@ -0,0 +1,668 @@
+        {this.translations.text.showingPage
+          .replace(ACTUAL_PAGE, this.actualPage?.toString())
+          .replace(
+            TOTAL_PAGES,
+            (this.totalPages > 0 && this.totalPages?.toString()) ||
+              this.translations.text.unknownPages
+          )}
+      </span>
+    </div>
+  );
+
+  #renderNavigationGoTo = () => (
+    <div part={tokenMap({ [PAGINATOR_PARTS_DICTIONARY.GO_TO]: true })}>
+      {this.translations.text.page}
+      <input
+        part={tokenMap({ [PAGINATOR_PARTS_DICTIONARY.GO_TO__INPUT]: true })}
+        type="number"
+        min="1"
+        max={this.totalPages}
+        value={this.actualPage}
+        onKeyDown={this.#handleGoToOnKeyDown}
+        onBlur={this.#handleGoToOnBlur}
+      />
+      {this.translations.text.of}
+      {this.translations.text.of &&
+        ((this.totalPages > 0 && this.totalPages?.toString()) ||
+          this.translations.text.unknownPages)}
+    </div>
+  );
+
+  render() {
+    // Array with controls for mapping
+    const controls = [
+      {
+        name: "itemsPerPage",
+        render: () => this.#renderItemsPerPage(),
+        show: this.showItemsPerPage
+      },
+      {
+        name: "itemsPerPageInfo",
+        render: () => this.#renderItemsPerPageInfo(),
+        show: this.showItemsPerPageInfo
+      },
+      {
+        name: "navigationControls",
+        render: () => this.#renderNavigationControls(),
+        show: this.showNavigationControls
+      },
+      {
+        name: "navigationGoTo",
+        render: () => this.#renderNavigationGoTo(),
+        show: this.showNavigationGoTo
+      },
+      {
+        name: "navigationControlsInfo",
+        render: () => this.#renderNavigationControlsInfo(),
+        show: this.showNavigationControlsInfo
+      },
+      {
+        name: "firstControl",
+        render: () => this.#renderFirstControl(),
+        show: this.showFirstControl
+      },
+      {
+        name: "lastControl",
+        render: () => this.#renderLastControl(),
+        show: this.showLastControl
+      },
+      {
+        name: "prevControl",
+        render: () => this.#renderPrevControl(),
+        show: this.showPrevControl
+      },
+      {
+        name: "nextControl",
+        render: () => this.#renderNextControl(),
+        show: this.showNextControl
+      }
+    ];
+
+    // Filter controls that has to be shown
+    const visibleControls = controls.filter(control => control.show);
+
+    // Sort visible controls regarding their order value
+    visibleControls.sort((a, b) => {
+      const orderA =
+        this.order[a.name as keyof ControlsOrder] ?? Number.MAX_SAFE_INTEGER;
+      const orderB =
+        this.order[b.name as keyof ControlsOrder] ?? Number.MAX_SAFE_INTEGER;
+      return orderA - orderB;
+    });
+
+    return (
+      <Host part={tokenMap({ [PAGINATOR_PARTS_DICTIONARY.CONTROLS]: true })}>
+        {visibleControls.map(control => control.render())}
+      </Host>
```

Comment:
We don't need to set a part on the Host element. This element can be styled via a class. For example:

```tsx
<ch-paginator-render class="paginator"></ch-paginator-render>
```

```scss
.paginator {
  background-color: #c0ffee;
}
```

```suggestion
      <Host>
        {visibleControls.map(control => control.render())}
      </Host>
```

Reply:
done